### ISSUE TITLE
[ReferenceError: Can't find variable: btoa]

### ISSUE THREAD

**Issue by sydimthi** [heroiclabs/nakama-js#85]
# React-native [ReferenceError: Can't find variable: btoa]
Hi there,
             I'm using **@heroiclabs/nakama-js(^2.1.3")**  in **react-native(^0.63.2)**. And this is the first time I'm using nakama in react native. 

My issue is while I'm calling,
```javascript
 client.authenticateCustom(sessionToken, create, user_id );
```
I'm getting error, 
**[ReferenceError: Can't find variable: btoa]**
![Screenshot 2021-01-17 025442](https://user-images.githubusercontent.com/34239669/104823623-7dcbf780-5871-11eb-9232-da0fc2d3d2e7.png)

=====================================================================================

**If I tooked build for android then it crashed my application, I'm getting like this -->**
![Screenshot 2021-01-17 025247](https://user-images.githubusercontent.com/34239669/104823617-6ee54500-5871-11eb-9c10-3f5d2a5f2066.png)

=====================================================================================

**But if I debugged through browser then it get success.**
![WhatsApp Image 2021-01-17 at 3 00 12 AM](https://user-images.githubusercontent.com/34239669/104823688-05196b00-5872-11eb-8551-f91c9cd57dfc.jpeg)

I tried it in **react-js** then it works perfectly.

=====================================================================================

Now I need a solution for this *[ReferenceError: Can't find variable: btoa]* . Please fix it ASAP.

**Comment by sydimthi** [comment]
# The global import of base64 in the **app.js** or **index.js** resolved this problem.
### Step 1
`npm i base-64`
### Step 2
#### paste below codes in **app.js** or **index.js**
```javascript
 import {decode, encode} from 'base-64'

if (!global.btoa) {  global.btoa = encode }

if (!global.atob) { global.atob = decode } 
```
### Finally, Works perfectly 👍

**Comment by lugehorsam** [comment]
For posterity, this will be fixed in nakama-js 2.2 so that you don't need to apply this money patch.

**Comment by maxbbb** [comment]
Just for reference I experienced this same issue on nakama-js 2.3.0, so it does not appear that 2.1.4 fixed it. The same global override worked for me as well.
